airflow/providers/teradata/example_dags/example_teradata_to_teradata_transfer_operator.py

```diff
@@ -0,0 +1,158 @@
+#
```

Comment:
example dags should be written as system tests and be stored in the system test directory

Reply:
example dags are removed